### ISSUE TITLE
Moving widgets inside a dashboard has become less smooth in RC

### ISSUE THREAD

**Issue by madalincm** [mozilla/redash#958]
### Steps to reproduce:
1. Open a dashboard
2. Click the options menu button from the top right corner
3. Click on the edit button
4. Try to move/scale the widgets/text boxed

### Expected results:
The process is smooth, the widgets can easily be moved/scaled

### Actual results:
The whole process is not smooth, when the user tries to move the widgets the move is jittery

### Notes
Verified in FF66 win7x64 Issue can be reproduced in STMO-RC
Please view this screencast: 
![dashboardEdit](https://user-images.githubusercontent.com/15685441/58180039-0ddb2780-7cb2-11e9-83db-5a2724d5b1eb.gif)

**Comment by emtwo** [comment]
My assessment here is that there is one issue I can consistently reproduce. It was that dragging a widget from high to low does not swap the widget with the one below it (it works if you drag the lower one up). This actually seems to be an issue in release too.

It looks like this specific issue is fixed upstream as of recently (May 16 - https://github.com/getredash/redash/commit/606cf12e743c4b6102532e1d8aef70b391c8695b) which we did not pull in to our merge.

I tried to cherry pick the fix but it doesn't apply smoothly as it's quite a large change.

Maybe we can hold off on this fix until the next merge?
